docs/src/index.md

```diff
@@ -2,7 +2,12 @@
 
 The purpose of the [`BinaryBuilder.jl`](https://github.com/JuliaPackaging/BinaryBuilder.jl) Julia package is to provide a system for compiling 3rd-party binary dependencies that should work anywhere the [official Julia distribution](https://julialang.org/downloads) does.  In particular, using this package you will be able to compile your large pre-existing codebases of C, C++, Fortran, Rust, Go, etc... software into binaries that can be downloaded and loaded/run on a very wide range of machines.  As it is difficult (and often expensive) to natively compile software packages across the growing number of platforms that this package will need to support, we focus on providing a set of Linux-hosted cross-compilers.  This package will therefore set up an environment to perform cross-compilation for all of the major platforms, and will do its best to make the compilation process as painless as possible.
 
-Note that at this time, BinaryBuilder itself runs on Linux `x86_64` and macOS `x86_64` systems only, with Windows support under active development.  On macOS and Windows, you must have `docker` installed as the backing virtualization engine.  Note that Docker Desktop is the recommended version; if you have Docker Machine installed it may not work correctly or may need additional configuration.
+!!! warning "BinaryBuilder only runs on x86_64 systems"
+    At this time, BinaryBuilder itself runs on Linux `x86_64` and macOS `x86_64` systems only,
+    with Windows support under active development. Explicitly, this means that BinaryBuilder will **not** run on the newer Apple Silicon 'M1' Macs.
```

Comment:
Well, it works with an x86_64 build of Julia on the M1

Reply:
Does it if docker is running natively?

Reply:
(checking myself, ignore above)

Reply:
When I run the following with Julia 1.7 & BinaryBuilder 1.5.1 via Rosetta:
```bash
BINARYBUILDER_AUTOMATIC_APPLE=true julia build_tarballs.jl --debug --verbose aarch64-apple-darwin
```

I get:

```julia
[ Info: Docker base image already exists, skipping import...
 Downloading artifact: PlatformSupport-aarch64-apple-darwin20.v2021.8.10.x86_64-linux-musl.unpacked
[ Info: Unpacking /var/folders/7c/hm96b6vx3r9dc4_s5f6y9m1r0000gn/T/jl_9Ba9Rf-download.gz into /Users/jpslewis/.julia/artifacts/jl_3ZvYJk...
┌ Error: Tree Hash Mismatch!
│   Expected git-tree-sha1:   e09f78a42f59ebfe2f7ea979afc8b3567126de59
│   Calculated git-tree-sha1: dcae944b39f6c9026aa8a6d710084d54b19ac43b
  Downloaded artifact: PlatformSupport-aarch64-apple-darwin20.v2021.8.10.x86_64-linux-musl.unpacked
 Downloading artifact: PlatformSupport-aarch64-apple-darwin20.v2021.8.10.x86_64-linux-musl.unpacked
[ Info: No hash cache found
[ Info: Calculated hash 4256776d422edd94ea420c7dcaf09f2b29df185be8e9e218528e66dd72c80632 for file /var/folders/7c/hm96b6vx3r9dc4_s5f6y9m1r0000gn/T/jl_AFSvpa-download.gz
[ Info: Unpacking /var/folders/7c/hm96b6vx3r9dc4_s5f6y9m1r0000gn/T/jl_AFSvpa-download.gz into /Users/jpslewis/.julia/artifacts/jl_EKJYiR...
┌ Error: Tree Hash Mismatch!
│   Expected git-tree-sha1:   e09f78a42f59ebfe2f7ea979afc8b3567126de59
│   Calculated git-tree-sha1: dcae944b39f6c9026aa8a6d710084d54b19ac43b
  Downloaded artifact: PlatformSupport-aarch64-apple-darwin20.v2021.8.10.x86_64-linux-musl.unpacked
ERROR: LoadError: Unable to automatically install 'PlatformSupport-aarch64-apple-darwin20.v2021.8.10.x86_64-linux-musl.unpacked' from '/Users/jpslewis/.julia/packages/BinaryBuilderBase/khVxP/Artifacts.toml'
Stacktrace:
  [1] error(s::String)
    @ Base ./error.jl:33
  [2] ensure_artifact_installed(name::String, meta::Dict{String, Any}, artifacts_toml::String; platform::Platform, verbose::Bool, quiet_download::Bool, io::Base.TTY)
    @ Pkg.Artifacts /Applications/Julia-1.7 2.app/Contents/Resources/julia/share/julia/stdlib/v1.7/Pkg/src/Artifacts.jl:441
  [3] ensure_artifact_installed(name::String, artifacts_toml::String; platform::Platform, pkg_uuid::Nothing, verbose::Bool, quiet_download::Bool, io::Base.TTY)
    @ Pkg.Artifacts /Applications/Julia-1.7 2.app/Contents/Resources/julia/share/julia/stdlib/v1.7/Pkg/src/Artifacts.jl:404
  [4] mount(cs::BinaryBuilderBase.CompilerShard, build_prefix::String; verbose::Bool)
    @ BinaryBuilderBase ~/.julia/packages/BinaryBuilderBase/khVxP/src/Rootfs.jl:258
  [5] #41
```

`x86_64-linux-gnu` builds work without a problem. For `x86_64-apple-darwin` I get the same error as above. Does your toolchain run through?

Reply:
What version of Julia is that?

Reply:
Julia 1.7

Reply:
1.7.0 has a known bug with computation of tree hashes

Reply:
aha, ok, let me try 1.7.1

Reply:
1.7.0 was the issue.